docs-web/app-protect/configuration.md

```diff
@@ -18,6 +18,8 @@ You can define App Protect policies for your Ingress resources by creating an `A
 
  > **Note**: The fields `policy.signature-requirements[].minRevisionDatetime` and `policy.signature-requirements[].maxRevisionDatetime` are not currently supported. 
 
+ > **Note**: [The Advanced gRPC Protection for Unary Traffic](/nginx-app-protect/configuration/#advanced-grpc-protection-for-unary-traffic) is not currently supported.
```

Comment:
this points to https://docs.nginx.com/nginx-app-protect/configuration/#advanced-grpc-protection-for-unary-traffic